environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+	"sync"
+	"github.com/fsouza/go-dockerclient"
+	"github.com/minio/minio-go"
+)
+
+type Container struct {
+	ID           string
+	statsChannel chan *docker.Stats
+	doneChannel chan bool
+	stopChannel chan bool
```

Comment:
I did it to avoid possible confusion, like having a routine stop and start immediately after, and then get stopped while leaving another hanging. Unlikely scenario, but you never know.

I'll change it to a single channel if you think that's not a problem.